ui/components/app/collectibles-items/collectibles-items.js

```diff
@@ -92,7 +92,12 @@ export default function CollectiblesItems({
     collectionName,
   ) => {
     if (isPreviouslyOwnedCollection) {
```

Comment:
I think we can get rid of this whole block:

```
if (isPreviouslyOwnedCollection) {
      return null;
      return (
        <img
          src={collectionImage}
          className="collectibles-items__collection-image"
        />
      );
    }
```

... since it matches the block below it.